src/styles/card.scss

```diff
@@ -0,0 +1,99 @@
+@import './variables';
+
+.card {
+  width: 200px;
+  flex-shrink: 0;
+  background-color: $white;
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid $gray-elements;
+  display: flex;
+  flex-direction: column;
+
+  transition: all 300ms ease-in-out;
+
```

Comment:
Can't agree here: instead of having margins for image && content you should use padding for the card. 

Same here: parent is responsible for positioning its children.